lib/glific/third_party/kaapi/api_client.ex

```diff
@@ -38,9 +38,11 @@ defmodule Glific.ThirdParty.Kaapi.ApiClient do
         body
       end
 
+    opts = [adapter: [recv_timeout: 30_000]]
+
```

Comment:
doubt: @shijithkjayan are we sure 30s would be enough? Since we’re not sure how much time Kaapi can take. Ideally, it shouldn’t take that long.